src/Microsoft.Azure.WebJobs/AutoResolveValue.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Description
+{
+    /// <summary>
+    /// Specify the default behavior for an <see cref="AutoResolveAttribute"/> property when it has an empty value. 
+    /// These only apply if the property is empty. They don't apply if the property is an expression that evaluates to empty. 
+    /// </summary>
+    public enum AutoResolveValue
```

Comment:
Behaviorally, it makes sense. AppSetting specifies a string; AutoResolve specifies a fixed policy. 
So maybe AutoResolve.Default needs a better name. 



---
In reply to: [114159831](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r114159831) [](ancestors = 114159831)